src/display-x11.cc

```diff
@@ -374,266 +377,7 @@ bool display_output_x11::main_loop_wait(double t) {
-
-            int border_total = get_border_total();
-
-            text_width = window.width - 2 * border_total;
-            text_height = window.height - 2 * border_total;
-            int mw = this->dpi_scale(maximum_width.get(*state));
-            if (text_width > mw && mw > 0) { text_width = mw; }
-          }
-
-          /* if position isn't what expected, set fixed pos
-           * total_updates avoids setting fixed_pos when window
-           * is set to weird locations when started */
-          /* // this is broken
-          if (total_updates >= 2 && !fixed_pos
-              && (window.x != ev.xconfigure.x
-              || window.y != ev.xconfigure.y)
-              && (ev.xconfigure.x != 0
-              || ev.xconfigure.y != 0)) {
-            fixed_pos = 1;
-          } */
-        }
-        continue;
-
-      case ButtonPress:
-#ifdef BUILD_MOUSE_EVENTS
-      {
-        modifier_state_t mods = x11_modifier_state(ev.xbutton.state);
-        if (4 <= ev.xbutton.button && ev.xbutton.button <= 7) {
-          scroll_direction_t direction =
-              x11_scroll_direction(ev.xbutton.button);
-          consumed = llua_mouse_hook(
-              mouse_scroll_event(ev.xbutton.x, ev.xbutton.y, ev.xbutton.x_root,
-                                 ev.xbutton.y_root, direction, mods));
-        } else {
-          mouse_button_t button = x11_mouse_button_code(ev.xbutton.button);
-          consumed = llua_mouse_hook(mouse_button_event(
-              mouse_event_t::MOUSE_PRESS, ev.xbutton.x, ev.xbutton.y,
-              ev.xbutton.x_root, ev.xbutton.y_root, button, mods));
-        }
-      }
-#endif /* BUILD_MOUSE_EVENTS */
-        if (own_window.get(*state)) {
-          /* if an ordinary window with decorations */
-          if ((own_window_type.get(*state) == TYPE_NORMAL &&
-               !TEST_HINT(own_window_hints.get(*state), HINT_UNDECORATED)) ||
-              own_window_type.get(*state) == TYPE_DESKTOP) {
-            /* allow conky to hold input focus. */
-            break;
-          }
-        }
-        break;
-
-      case ButtonRelease:
-#ifdef BUILD_MOUSE_EVENTS
-        /* don't report scroll release events */
-        if (4 > ev.xbutton.button || ev.xbutton.button > 7) {
-          modifier_state_t mods = x11_modifier_state(ev.xbutton.state);
-          mouse_button_t button = x11_mouse_button_code(ev.xbutton.button);
-          consumed = llua_mouse_hook(mouse_button_event(
-              mouse_event_t::MOUSE_RELEASE, ev.xbutton.x, ev.xbutton.y,
-              ev.xbutton.x_root, ev.xbutton.y_root, button, mods));
-        }
-#endif /* BUILD_MOUSE_EVENTS */
-        if (own_window.get(*state)) {
-          /* if an ordinary window with decorations */
-          if ((own_window_type.get(*state) == TYPE_NORMAL) &&
-              !TEST_HINT(own_window_hints.get(*state), HINT_UNDECORATED)) {
-            /* allow conky to hold input focus. */
-            break;
-          }
-        }
-        break;
```

Comment:
These consecutive breaks made no sense to me, so I tracked down that the window type check was intended to avoid propagation of events (inner break) in the [original code](https://github.com/Caellian/conky/commit/7fbdfbd4d45a99b06def7065d66e02337323066d#diff-5da03e30dd643b06945321a0509384d16ad7783650cc3ca2f967dad551858cf7L466).

Over several commits, I seem to have made it do nothing.